klaxon/src/main/kotlin/com/beust/klaxon/KlaxonJson.kt

```diff
@@ -15,8 +14,11 @@ interface KlaxonJson {
     // we need this as now JsonArray<T> is List<T>
     fun <T> array(subArray : JsonArray<T>) : JsonArray<JsonArray<T>> = JsonArray(listOf(subArray))
 
+    fun obj(args: Collection<Pair<String, *>>): JsonObject =
+            JsonObject(args.toMap(LinkedHashMap()).mapValues { convert(it.value) })
```

Comment:
By adding this overload it allows API consumers to prevent an expensive array copy.